start/software/compiling-applications.md

```diff
@@ -1,57 +1,118 @@
+your `LD_LIBRARY_PATH` as a step in your job's executable bash script before executing 
+your software.
 
-## Compilation Server
+The [distributed environment modules system](https://support.opensciencegrid.org/support/solutions/articles/12000048518) 
+available on OSG Connect also provides several commonly used software libraries 
+(lapack, atlas, hdf5, netcdf, etc.) that can be used for your software compilation and 
+execution. The appropriate modules should be loaded before performing your software 
+compilation. The process of loading a module will modify all appropriate 
+environment variables (e.g. `CPATH` and `LIBRARY_PATH`) for you. If you do use modules 
+from the modules system, you will need to modify your job scripts to load the appropriate 
+modules before running you software.
 
-Regardless of whether you statically or dynamically compile your application,
-you should compile your application on your assigned login node.  This will ensure
+## Perform your compilation
+
+Software compilation is easiest to perform interactively, and OSG Connect users are 
+welcome to compile software directly on their assigned login node. This will ensure
 that your application is built on an environment that is similar to the majority
-of the compute nodes on OSG.  In addition, you will need to add the following
-require to your HTCondor submit file: `(OpSysAndVer =?= "SL7") || (OpSysAndVer
-=?= "RHEL7") || (OpSysAndVer =?= "Centos6")` to make sure that your binaries run 
-systems that have compatible linux distributions.
-
-## Libraries
-
-The distributed environment modules system provides several commonly used
-scientific libraries (lapack, atlas, hdf5, netcdf, etc.) that your application
-may need to link to.  In order to link against a library provided by the
-module system, just load the appropriate module (e.g. `hdf5/1.8.12`).  Doing so
-will set the `CPATH` and `LIBRARY_PATH` so that gcc will be able to find the
-includes and link files for that library.  If your build system does not pick up
-the appropriate you can look at the `CPATH` and `LIBRARY_PATH` variables to find
-the paths used and incorporate that into your build process or you can contact
-user support for assistance.
-
-Finally, if you do use modules from the modules system, you will need to modify
-your job scripts to load the appropriate modules before running you application.
-
-## Assistance
-If you have questions or need assistance, please contact <support@osgconnect.net>.
+of the compute nodes on OSG. Because OSG Connect login nodes currently use the 
+Red Hat Enterprise Linux 7 operating system, your software will, generally, only be 
+compatible for execution on RHEL 7 or similar operating systems. You can use the 
+`requirements` statement of your HTCondor submit file to direct your jobs to execute 
+nodes with specific operating systems, for instance:
+
+	requirements = (OpSysAndVer =?= "SL7") || (OpSysAndVer=?= "RHEL7") || (OpSysAndVer =?= "Centos7")
```

Comment:
We don't normally recommend all three of these - can we get by with just one of them? Maybe a question for @rynge

Reply:
I carried this over from the previous guide

Reply:
has been addresses in commit a4c4e8256c3e45b2bb5da4a95acd0929692c3d4a